src/scala/com/github/johnynek/bazel_deps/Normalizer.scala

```diff
@@ -55,14 +57,26 @@ object Normalizer {
           table(n).collect { case (_, Right(v)) => v }.toSet.size > 1
       }
 
-      def disambiguate(node: Node): Table =
+      def disambiguate(node: Node): Table = disambiguateHelper(node, new mutable.LinkedHashSet)
+
+      def disambiguateHelper(node: Node, visited: mutable.LinkedHashSet[Node]): Table = {
```

Comment:
Can we just use Set here and not anything mutable?

Reply:
Note Set require no imports.

Reply:
Yep, done. I'd originally thought that we'd want to keep the path that caused the cycle (e.g. A -> B -> C -> A), but I changed my mind at some point while I was stepping through it and forgot to swap the LinkedHashSet out.